pilot-app/src/components/form/wimForm.vue

```diff
@@ -0,0 +1,316 @@
+﻿<template>
+    <section>
+        <table class="formTable">
+            <tbody>
+                <tr v-for="row in rows" :key="row.id">
+                    <td v-if="row.isButtonRow">
+                        <component
+                            v-for="field in row.fields"
+                            :key="field.propertyName"
+                            :is="field.vueType"
+                            :field="field"
+                            @onclick="onButtonClick"
+                        ></component>
+                    </td>
+                    <template v-else v-for="field in row.fields">
+                        <component
+                            v-if="field.vueType === 'wimSection'"
+                            :key="field.propertyName"
+                            :is="field.vueType"
+                            :field="field"
+                            v-model="field.value"
+                        ></component>
+                        <template v-else v-show="showField(field)">
+                            <th :class="expressCell(field.expression)" colspan="1">
+                                <label
+                                    v-if="!hideLabelForType(field.vueType)"
+                                    :for="field.propertyName"
+                                    :title="field.title"
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 18 spaces but found 36 spaces.](https://app.codacy.com/gh/Supershift/Sushi.Mediakiwi.UI/pullRequest?prid=8170341)
```suggestion
                                    v-html="field.title"
```